docs/migration/migrate_8_0.asciidoc

```diff
@@ -50,41 +50,41 @@ for example, `logstash-*`.
 
 [float]
 ==== Default logging timezone is now the system's timezone
-*Details:* In prior releases the timezone used in logs defaulted to UTC.  We now use the host machine's timezone by default.
+*Details:* In prior releases the timezone used in logs defaulted to UTC. We now use the host machine's timezone by default.
 
-*Impact:* To restore the previous behavior, in kibana.yml use the pattern layout, with a date modifier:
+*Impact:* To restore the previous behavior, in kibana.yml use the pattern layout, with a <<date-format, date modifier>>:
```

Comment:
These breaking changes are also published in the Installation and Upgrade Guide (https://www.elastic.co/guide/en/elastic-stack/master/kibana-breaking-changes.html), so you can't use internal links like this (unless the target *also* exists in the Installation and Upgrade Guide). Instead, use an external link:

```suggestion
*Impact:* To restore the previous behavior, in kibana.yml use the pattern layout, with a {kibana-ref}/date-format.html[date modifier]:
```

Reply:
Ah, thank you!

Reply:
@lcawl It's going to be easier to add these links once the main logging documentation is merged. Thanks for the guidance!